back-end/tests/models/user-quests.test.js

```diff
@@ -0,0 +1,54 @@
+const User = require("../../db/models/users");
+const Quest = require("../../db/models/quests");
+const faker = require("faker");
+// sequelizeClient.sync({ force: true, match: /_test$/ });
+
+describe("user relationships", () => {
+  it("can fetch the related quests", async () => {
+    // data creation
+    const villager = User.create({
+      username: faker.internet.userName(),
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      email: faker.internet.email(),
+      password: faker.internet.password(),
+      avatar: faker.internet.avatar(),
+      adventurer: false,
+      bio: faker.lorem.paragraph(),
+    });
+
+    const adventurer = User.create({
```

Comment:
```suggestion
    const adventurer = await User.create({
```
and I will let you deal with the other `create` calls returning a promise (instead of writing a PR comment).